signup.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>Sign up</title>
+<meta charset="utf-8">
+<link href="css/style2.css" rel='stylesheet' type='text/css' />
```

Comment:
Signup and Login should have common CSS considering they are quite similar pages.